src/main/resources/apache-v2-license.txt

```diff
@@ -1,202 +0,0 @@
-
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
-
-   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
@andrewoelfing It seems some files are deleted by mistake ? Can you revert these ?

Reply:
Hello @sancar, I'm not aware of deleting these files. But I will revert these changes.